frontend/freebase2wikidata.js

```diff
@@ -260,8 +260,85 @@ $(function() {
     var title = mw.config.get('wgTitle');
     return qidRegEx.test(title) ? title : false;
   }
-  
-  (function generateNav() { 
+
+    ps.openNav = function openNav(itemLabel, propertyLabel, propertyValue, referenceURL, buttons) {
+        $('#myNav').width('100%');
+
+        var blackboard = $('#blackboard');
```

Comment:
Instead of building your own window system you could maybe rely on https://doc.wikimedia.org/oojs-ui/master/js/#!/api/OO.ui.ProcessDialog that seems to fit quite well this use case.

PrimarySources is already using it for the PrimarySources list feature and there is already a windowManager created at the end of this file.

Reply:
This is a great suggestion. I used a simple div as a window system as it seemed the fastest and easiest way to me. I will try to look into the OO.ui library.